BaseTools/Plugin/HostBasedUnitTestRunner/HostBasedUnitTestRunner.py

```diff
@@ -52,6 +52,12 @@ def do_post_build(self, thebuilder):
 
         failure_count = 0
 
+        # Do not catch exceptions in gtest so they are handled by address sanitizer
+        shell_env.set_shell_var('GTEST_CATCH_EXCEPTIONS', '0')
+
+        # Disable address sanitizer memory leak detection

```

Comment:
Are there plans to enable this in the future?

Reply:
Perhaps.  Memory leaks are bad, but unless the system runs out of memory, they do not cause any unexpected or undefined behaviors.  I have also observed that the unit test harnesses themselves leak memory, so it is hard to enable this and avoid false positives.  There are also some FW init flows that allocate buffers for OS tables that are never freed which would be another source of false positives.  We would have to find a way to enable this on code under test that we know should always balance the alloc/free calls.